config/config.yml

```diff
@@ -1,5 +1,6 @@
 ---
-debug: # true for debugging
+debug: debug # fatal, error, warn, info, debug; defaults to 'fatal'
+logfile: stderr # stderr, stdout, or path; defaults to log/nailed.log
```

Comment:
maybe you misunderstood me here. in the config.yml we have only the keys and the possible values **commented out**. the keys you added are right, only the values shouldn't be in there, as the user writes those in the file

so `debug:` and `logfile:` should stay and `debug` and `stderr` goes out